practice-one/src/App.jsx

```diff
@@ -1,11 +1,16 @@
-import './App.css';
+import React from 'react';
+import Button from './Components/Common/Buttons';
 
 function App() {
+  const handleClick = () => {
+    console.log('Button clicked!');
+  };
+
   return (
-    <div className="App">
-      <h1 className="text-3xl text-purple-400 font-bold underline">
-        Hello world!
-      </h1>
+    <div>
+      <Button text="login" variant="primary" onClick={handleClick} />
```

Comment:
Please break line for each property. Update ALL similar places

Reply:
Thanks! Done